trans.py

```diff
@@ -16,80 +35,85 @@ def find_header(counter, width):
     return -1, -1
 
 def read_pack(name_dfr):
-    '''To read the input info from the Data Format Record'''
+    '''To read the input info from the Data Format Record (DFR)'''
     with open("infos.txt", "r", encoding="UTF-8") as info_read:
```

Comment:
Why are you reading the configuration file twice?

Reply:
Pylint doesn't let me have more than 15 variables per function, so I have to use one the functions to read how many files are we gonna be dealing with (read_howmany_names after renaming it) and another one to actually get the input info to interpret the DFR file from NASA (read_dfr after renaming).